portal-kernel/test/unit/com/liferay/portal/kernel/util/TextFormatterTest.java

```diff
@@ -136,39 +138,39 @@ public void testformatStorageSizeOneB() throws Exception {
 		long bytes = 1;
 
 		Assert.assertEquals(
-			"1 B", TextFormatter.formatStorageSize(bytes, LocaleUtil.SPAIN));
+			"1 B", LanguageUtil.formatStorageSize(bytes, LocaleUtil.SPAIN));
```

Comment:
@jeffwu0724 The logic had been removed from TextFormatterTest, thus you do not need to test the logic here, you have two options here:
1. Move the test logic to LanguageUtilTest
2. If corresponding test logic had been tested in LanguageUtilTest, then remove it directly.